edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/MetricsWorker.cs

```diff
@@ -40,7 +40,12 @@ public MetricsWorker(IMetricsScraper scraper, IMetricsStorage storage, IMetricsP
             this.uploader = Preconditions.CheckNotNull(uploader, nameof(uploader));
 
             this.metricFilter = new MetricTransformer()
-                .AddAllowedTags(new KeyValuePair<string, string>(MetricsConstants.MsTelemetry, true.ToString()))
+                .AddAllowedTags((MetricsConstants.MsTelemetry, true.ToString()))
+                .AddDisallowedTags(
```

Comment:
Is this how we make sure RQT only captures P90?

Reply:
yep